W-14101902-jms-connector-update-source-kt (#2384)
mulesoft/docs-connectors: jms/1.9/modules/ROOT/pages/jms-listener.adoc

Anypoint Connector for JMS (JMS Connector) *On New Message* source enables you to listen for and consume new messages as they arrive at the specified destination.

[Note]

> If you use JMS Connector with Oracle JMS, the *On New Message* source receives new messages with a delay of 1 to 10 seconds in between based on the frequency of messages. Control this behavior using `-Doracle.jms.maxSleepTime=xxxx`. For further details, see https://forums.oracle.com/ords/apexds/post/jms-client-receives-aq-messages-asynchronously-with-a-delay-1125[JMS client receives AQ messages asynchronously with a delay of 15 seconds^] 

Reviewer: ```suggestion
If you use JMS Connector with Oracle JMS, the *On New Message* source receives new messages with a delay of 1 to 10 seconds, based on the frequency of the messages. Control this behavior by using `-Doracle.jms.maxSleepTime=xxxx`. For further details, see https://forums.oracle.com/ords/apexds/post/jms-client-receives-aq-messages-asynchronously-with-a-delay-1125[JMS client receives AQ messages asynchronously with a delay of 15 seconds^]. 
```
